addons/spreadsheet/static/src/global_filters/helpers.js

```diff
@@ -229,6 +229,60 @@ const FILTERS_BEHAVIORS = {
         },
     ],
     boolean: [SET_OPERATORS_BEHAVIORS],
+    numeric: [
+        {
+            operators: ["=", "!=", ">", "<"],
```

Comment:
I don't know which operator should be the default one (the first operator of the first object in the `numeric` list).
"is equal to" is probably the worse as I think checking a single specific value is quite specific.
between maybe ? could you ask François ?